docs/imports/rock-the-vote.md

```diff
@@ -0,0 +1,120 @@
+
+So, the Alpha sends their page to a Beta and they register. The Alpha's referral links look like this: https://vote.dosomething.org/member-drive?userId={userId}&r=user:{userId},campaign:{campaignID},campaignRunID={campaignRunID},source=web,source_details=onlinedrivereferral,referral=true
+
+We've added `referral=true` to the link so that we can know to not attribute the registration to the NS ID that is present in the URL. In this case, this NS ID is the referrer and not the registrant.
+
+If the referral column has `referral=true` in it, then proceed with the logic with dealing w/ non-member registrants above.
+
+### How to count these as impact
+
+Based on the above statuses, some should be counted as a RB and some should not. This determination was made by the executive team and allows us to report internally progress towards the organization's report back goal. Here's what counts as a reportback from the RTV export:
+
+- `register-form`
+- `register-OVR`
+
+Note: `register-form` and `register-OVR` are the only statuses that count as _registrations_.
+
+Rogue will NOT store this information, but will return a derived value in the JSON response when the voter registration post is created or read that holds this information. The logic to determine this is as follows:
+
+```php
+if (in_array($rogueStatus, ['confirmed', 'register-form', 'register-OVR'])) {
+    $reportbackStatus = 'T';
+} else {
+    $reportbackStatus = 'F';
+}
+```
+
+## Notes
+
+- Data uses the post `details` to determine `source` and `source_detail` used in voter registration reporting.
+- The `submission_created_at` date is when the importer ran. Details about when the registration was created/updated are in the `source_details`.
+- All of these signups will have a `source` of `importer-client` (this is how messaging is suppressed in C.io)
+- In early iterations of the import, the month that the registration came in would inform the `action` column (e.g., february-2018-rockthevote)
+- In early iterations of the import, we would pass Campaign/Run IDs as parameters within the referral code to use when upsert a  `voter-reg` post.
+- If a user shares their UTM'ed URL with other people, there could be duplicate referral codes but associated with different registrants:
+  See a [screenshot](https://cl.ly/0v210N283y2X) of what this data looks like (note: the user depicted in this spreadsheet is fake.)
+
+## Open Questions
+
+1. How can we distinguish between TV and RTV import? Do we need to?
```

Comment:
Seems like this is not an issue.

Reply:
Hmm yeah since we don't use TV anymore I think this is a non-issue but maybe @ngjo has other thoughts?

Reply:
We do save `rock-the-vote` to the source detail currently, but ideally we want to change this to a stringified object to track referring user ID if the `referral` parameter is set  (addressed in [#171159899](https://www.pivotaltracker.com/story/show/171159899)). Not sure if we ever set this to `turbovote` for the old import

Reply:
Yea, I don't think is an issue anymore, but might be worth connecting with Tej on what the backup would be if RTV went down or anything in case this would be something we'd still want to be able to distinguish.

If anything changes on the `source_detail`s, please loop in Anthony/Tej/Data because Customer.io is looking at `source_detail`s and Tej/data is probably querying off of `source_detail`s

Reply:
I left a question in the inventory doc about the TV backup!  Definitely be working slowly and cross-functionally through any new `source_detail` changes we make, if any.